boxoffice/static/js/views/router.js

```diff
@@ -3,30 +3,52 @@ import {IndexView} from './index.js';
 import {OrgView} from './org.js';
 import {ItemCollectionView} from './item_collection.js';
 import {OrderView} from './admin_order.js';
+import {DiscountPolicyView} from './admin_discount_policy.js';
 import {ReportView} from './admin_report.js';
 
 export const Router = Backbone.Router.extend({
   url_root: '/admin/',
   routes: {
     "": "index",
-    "o/:org": "org",
-    "ic/:icId": "item_collection",
-    "ic/:icId/orders": "order",
-    "ic/:icId/reports": "report"
+    "o/:org_name": "org",
+    "ic/:ic_id": "item_collection",
+    "ic/:ic_id/orders": "order",
+    "o/:org_name/discount_policy": 'discount_policy',
+    "o/:org_name/discount_policy?:params": 'discount_policy',
+    "ic/:ic_id/reports": "report"
   },
   index: function() {
-    IndexView.render();
+    //View is 'home', used for urlFor().
+    IndexView.render('index');
   },
-  org: function(org){
-    OrgView.render({name: org});
+  org: function(org_name) {
+    OrgView.render('view', {org_name});
   },
-  item_collection: function(icId){
-    ItemCollectionView.render({id: icId});
+  item_collection: function(ic_id) {
+    ItemCollectionView.render('dashboard', {ic_id});
```

Comment:
@vidya-ram The `render` calls need to be fixed, both at the caller and the callee, since the first argument isn't being used anymore.